framework/v2/guides/providers.md

```diff
@@ -68,3 +68,93 @@ Error example:
   message: 'Sender does not have sufficient balance.'
 }
 ```
+
+## Create your own provider
+
+We have some predefined providers so you can use out of the box but if you have some other way you want to connect to blockchain (e.g. you want to do raw transaction) you can write you own provider. This will show you the basic principle of how to create you own provider.
```

Comment:
We have some predefined providers you can use out of the box, but if you have an alternative way you want to connect to the blockchain (e.g. you want to do raw transaction) you can write your own provider. This will show you the basic principles of creating your own provider.